tests/e2e/tests/base/configure-and-run-analysis.test.ts

```
@@ -10,16 +10,22 @@ test.describe(`Configure extension and run analysis`, () => {
+
+  const CATEGORY_NAMES = ['Potential', 'Optional', 'Mandatory'] as const;
 
   test.beforeAll(async ({ testRepoData }) => {
     test.setTimeout(900000);
-    const repoInfo = testRepoData['coolstore'];
+    repoInfo = testRepoData['coolstore'];
     vscodeApp = await VSCode.open(repoInfo.repoUrl, repoInfo.repoName);
   });
 
-  test('Create Profile and Set Sources and targets', async ({ testRepoData }) => {
+  test('Create Profile and Set Sources and targets', async ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)